src/Bicep.Core.UnitTests/Utils/StringTestUtils.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.RegularExpressions;
+using System.Linq;
+using Bicep.Core.Parsing;
+
+namespace Bicep.Core.UnitTests.Utils
+{
+    public static class StringTestUtils
+    {
+        private static Regex LineIndentRegex { get; } = new Regex("^\\s*");
```

Comment:
Should we add below RegexOptions(RegexOptions.Compiled | RegexOptions.CultureInvariant) here?